Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -30,36 +220,48 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
             return
         }
 
-        tabTableView.snp_makeConstraints { make in
-            make.top.equalTo(toolbar.snp_bottom)
+        tableView.snp_makeConstraints { make in
+            make.top.equalTo(self.toolbar.snp_bottom)
             make.left.right.bottom.equalTo(self.view)
         }
     }
 
     func SELdidClickDone() {
-        dismissViewControllerAnimated(true, completion: nil)
+        presentingViewController!.dismissViewControllerAnimated(true, completion: nil)
+    }
+
+    func SELdidClickSettingsItem() {
+        let controller = SettingsViewController(nibName: "SettingsViewController", bundle: nil)
+        controller.profile = profile
+        presentViewController(controller, animated: true) { () -> Void in
+            // ?
+        }
```

Comment:
<code>completion: nil</code> instead of an empty callback with a mysterious question mark